bus/client/objects.go

```diff
@@ -49,29 +49,29 @@ func (c *Client) DeleteObject(ctx context.Context, bucket, path string, opts api
 	return
 }
 
-// ListOBjects lists objects in the given bucket.
-func (c *Client) ListObjects(ctx context.Context, bucket string, opts api.ListObjectOptions) (resp api.ObjectsListResponse, err error) {
-	err = c.c.WithContext(ctx).POST("/objects/list", api.ObjectsListRequest{
-		Bucket:  bucket,
-		Limit:   opts.Limit,
-		Prefix:  opts.Prefix,
-		Marker:  opts.Marker,
-		SortBy:  opts.SortBy,
-		SortDir: opts.SortDir,
-	}, &resp)
+// Objects returns the object at given path.
+func (c *Client) Object(ctx context.Context, bucket, key string, opts api.GetObjectOptions) (res api.Object, err error) {
```

Comment:
`path` vs `key`

Reply:
Ok that sent me down quite the rabbit hole since we were inconsistent all over the place.

I updated all occurrences of `Name` or `Path` when we refer to an object to `Key` same as S3. 
When we return a `object.EncryptionKey` that field is now called `EncryptionKey`.